st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -171,6 +172,12 @@ def run(self, action_parameters):
                 'traceback': trace
             }
 
+        parent_context = {
+            'execution_id': self.execution_id
+        }
+        if getattr(self.liveaction, 'context', None):
+            parent_context.update(self.liveaction.context)
```

Comment:
This would actually modify `self.liveaction.context` attribute in place if `self.liveaction.context` is defined.

I don't think that's desired (we would need to make a copy) :)